src/utils/DragAndDrop.js

```diff
@@ -398,6 +416,8 @@ define(function (require, exports, module) {
                 // Special-case .zip files, so we can offer to extract the contents
                 if(Path.extname(filename) === ".zip") {
                     handleZipFile(deferred, file, filename, buffer, encoding);
+                } else if(Path.extname(filename) === ".tar") {
```

Comment:
So we won't be able to handle `.tar.gz` files yet right? We shouldn't worry about gzipped files?